### PR TITLE
Introduce a wrapper for Google Group ID

### LINKED ISSUE #28: Unable to register `GoogleGroupPartRepository`
Now, when a `GoogleGroupPartRepository` is registered in a bounded context, the following exception is thrown:

`java.lang.IllegalStateException: Aggregates of the repository io.spine.users.google.c.group.GoogleGroupPartRepository neither handle commands nor react on events.`

Now the part only imports events. Probably, the design of the part should be changed.

### PR DESCRIPTION
**Pull request by dmytro-grankin**
The PR introduces `GoogleGroupId`.

Previously, the ID was represented as a `string`. But, it always looks better to have a wrapper.

Other changes:
- Redundant imports were removed from `.proto` files.
- Several issues in the field naming were fixed.

### CODE REVIEW

**Comment by dmytro-grankin** [comment]
@dmdashenkov PTAL.

File: users/src/main/proto/spine/users/google/c/group/events.proto [hunk 1]
```diff
@@ -55,7 +56,7 @@ message GoogleGroupCreated {
     GroupId id = 1;
 
     // An ID of the group at Google.
-    string google_id = 2 [(required) = true];
+    GoogleGroupId google_id = 2 [(required) = true, (valid) = true];
```
Comment (dmdashenkov) [inline users/src/main/proto/spine/users/google/c/group/events.proto]: I believe, `valid` implies `required`.

File: users/src/main/proto/spine/users/google/c/group/events.proto [hunk 2]
```diff
@@ -67,7 +68,7 @@ message GoogleGroupCreated {
     string description = 5;
 
     // A list of email aliases of the group.
-    repeated net.EmailAddress alialses = 6;
+    repeated net.EmailAddress aliases = 6;
```
Comment (dmdashenkov) [inline users/src/main/proto/spine/users/google/c/group/events.proto]: Please use the singular `alias`.

**Comment by dmytro-grankin** [comment]
@dmdashenkov PTAL.

**Pull request merged**